CollapseLauncher/Classes/GameManagement/GameVersion/BaseClass/GameVersionBase.cs

```diff
@@ -90,23 +90,12 @@ public string GameOutputLogName
             {
                 GameVersion? currentInstalled = GameVersionInstalled;
 
-                // Check if the pre_download_game property has value. If not, then return null
-                if (GameAPIProp.data.pre_download_game == null) return null;
                 // If no installation installed, then return null
                 if (currentInstalled == null) return null;
+                // Check if the pre_download_game property has value. If not, then return null
+                if (GameAPIProp.data.pre_download_game == null) return null;
 
-                // If has it, then check if the pre_download_game has diff.
-                // If null or no data, then return the latest one.
-                if (GameAPIProp.data.pre_download_game.diffs == null
-                 || GameAPIProp.data.pre_download_game.diffs.Count == 0) return TryGetNextVersionFromPkgFileName(GameAPIProp.data.pre_download_game.latest);
```

Comment:
As per this changes do not need ``TryGetNextVersionFromPkgFileName()`` method anymore, you can remove the method as well. The rest is looking good for me.

Reply:
Resolved as per latest commit